docs/dictionary/message/menuPick.lcdoc

```diff
@@ -68,6 +68,39 @@ menuHistory <property>.
 > <parameter> is the text that appears in the menu as displayed, not the
 > text of that line of the <button menu|menu button>.
 
+The menu item specification allows a tag to be specified using the form:
+
+    <label> [ '/' [ <accelerator> ] [ '|' <tag> ] ]
+    
+Note that the <tag> is optional as is the <accelerator> however, if you
+want an item with a tag but without the accelerator you need:
+
+    <label> '/' '|' <tag>
+  
+The <tag> must only be composed of characters from the ASCII character
+set.
```

Comment:
Hmm... that's a good question! I suspect not but I'll need to test to work it out.